src/sidebar/components/test/menu-item-test.js

```diff
@@ -150,4 +151,46 @@ describe('MenuItem', () => {
       'Submenu content'
     );
   });
+
+  it(
+    'should pass a11y checks',
+    checkAccessibility([
+      {
+        name: 'default',
+        content: () => (
+          <div role="menu">
```

Comment:
> So I'm assuming this is because we don't have the parent that defines a "role?"

You're right. axe-core has some checks related to this.

> I'm wondering in these cases if we should just do the a11y tests from the parent down

Yeah. In the case of `Menu` it probably would make sense to do more of a combined test that covers both `Menu` and `MenuItem` and just add an appropriate note in the `MenuItem` tests.